landoui/templates/partials/layout.html

```diff
@@ -13,7 +13,7 @@
 {% include "partials/navbar.html" %}
 <section class="hero is-light is-bold">
   <div class="container">
-    <p class="subtitle">{% block banner_text %}{% endblock %}</p>
+    <div class="subtitle"></div>
```

Comment:
An alternative would be just a colored stripe, which could be what you have done already (haven't seen it)